themes/default/content/registry/packages/gcp/installation-configuration.md

```diff
@@ -26,6 +26,35 @@ To provision resources with the Pulumi Google Cloud Provider, you need to have G
 
 If you are using Pulumi in a non-interactive setting (such as a CI/CD system) you will need to [configure and use a service account](/registry/packages/gcp/service-account) instead.
 
+### Dynamically generate credentials
+
+In addition to configuring the GCP provider locally, you also have the option to centralize your configurations using [Pulumi ESC (Environments, Secrets, and Configuration)](/docs/pulumi-cloud/esc/). Using this service will enable you to run Pulumi CLI commands with dynamically generated credentials, removing the need to configure and manage your credentials locally.
+
+To do this, you will need to complete the following steps:
+
+#### Configure OIDC between Pulumi and GCP
+
+Refer to the [Configuring OpenID Connect for GCP Guide](/docs/pulumi-cloud/oidc/gcp/) for the step-by-step process on how to do this.
+
```

Comment:
The previous sections include some additional reference to what the ESC environment should actually contain before they talk about importing the environment. I'm assuming this one is different because of the "Configuration" section below?

Reply:
This is a really good callout, yeah. There's significant structural variation between all of these docs (separate from this PR of course) that's probably not helping here.

Let's go ahead and get this merged, and I'll get an issue filed for applying some consistency here.

Reply:
Yes I was trying to find the best way to fit the content in with the existing content on each page but I agree that finding a way to make them more consistent would be best. I wasn't sure if there was a specific reason they were broken up that way. I will add this to my to-do for the quarter.